### PR TITLE
Document --strict deprecation

### PR DESCRIPTION
**Pull request by m-blaha**
It's a difference between dnf4 and dnf5 and we should forget to mention it in the docs.

### CODE REVIEW

File: CHANGES-DNF-5.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ options and implemented only for related commands
 Renaming boolean options to format `--<option>`, and `--no-<option>`
 `--nobest` -> `--no-best`. Proposing to keep `--nobest` as a deprecated option.
 
+### --strict deprecation
+`--strict` option is now deprecated. The problem with this option is that it does two things:
+
+1. it allows the solver to skip uninstallable packages to resolve depsolv problems
+2. it allowd dnf to skip unavailable packages (this is for `install` command mostly)
```
Comment (kontura) [inline CHANGES-DNF-5.md:56]: allowd -> allowed

File: CHANGES-DNF-5.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ options and implemented only for related commands
 Renaming boolean options to format `--<option>`, and `--no-<option>`
 `--nobest` -> `--no-best`. Proposing to keep `--nobest` as a deprecated option.
 
+### --strict deprecation
+`--strict` option is now deprecated. The problem with this option is that it does two things:
```
Comment (kontura) [inline CHANGES-DNF-5.md:53]: If I understand correctly I would maybe reword it a bit:
- isn't the `strict` option a config option? I can't find any `--strict` switch
- I would say if the strict option was disabled it allows those two things, currently to me the text suggests that if the option was enabled it allows those things.

**Review by kontura — CHANGES_REQUESTED** [review]

File: CHANGES-DNF-5.md [hunk 3]
```diff
@@ -49,6 +49,14 @@ options and implemented only for related commands
...
+
+1. it allows the solver to skip uninstallable packages to resolve depsolv problems
+2. it allowd dnf to skip unavailable packages (this is for `install` command mostly)
+
+The functionality is now splitted to two options - `--skip-broken` for the uninstallable packages and `--skip-unavailable` for packages not present in repositories.
```
Comment (kontura) [inline CHANGES-DNF-5.md:58]: I think that while `splitted` is a word just `split` would [be better](https://www.usingenglish.com/reference/irregular-verbs/split.html).

File: CHANGES-DNF-5.md (hunk 2 above)
Comment (m-blaha) [reply CHANGES-DNF-5.md]: Thanks for suggestions. I've tried to make it more clear.

**Review by kontura — APPROVED** [review]
LGTM
